articles/quickstart/webapp/java-ee/_includes/_setup.md

```diff
@@ -76,7 +76,7 @@ Your Java EE application needs some information in order to authenticate users w
 </env-entry>
 ```
 
-This information will be used to configure the Auth0 Java MVC SDK to allow authentication with users. To learn more about the SDK, including its various configuration options, see the [README](https://github.com/auth0/auth0-java-mvc-common/blob/master/README.md) of the SDK.
+This information will be used to configure the **auth0-java-mvc-commons** library to enable users to login to your application. To learn more about the library, including its various configuration options, see the [README](https://github.com/auth0/auth0-java-mvc-common/blob/master/README.md) of the library.
```

Comment:
Maybe "this libraries README" or "see the README for that library".  "see the README of the library" sounds awkward.